### ISSUE TITLE
Screen Reader Support Specification Adaptation - Bar Component

### ISSUE THREAD

**Issue by elenastoyanovaa** [SAP/ui5-webcomponents#2669]
**Is your feature request related to a problem? Please describe.**
We should analyse the existing screen reader support implementation, along with the UI5 SR Specification itself and Aria Best Practices and adapt them in a way that fit the ui5 web components.

**Describe the solution you'd like**
Analyse, create and discuss a proposal specification with @elenastoyanovaa for the bar component.

After the discussion, the approved screen reader support implementation proposal, should be documented in the accessibility status wiki of the component.

**Comment by ghristanov** [comment]
A small enhancement will be implemented with https://github.com/SAP/ui5-webcomponents/issues/3020.
